### ISSUE TITLE
Can't access options menu in multiselect with screen reader on Firefox

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#5093]
On Firefox using NVDA on Windows 10 and VoiceOver on macOS I was unable to navigate through the options list and make a selection. That said -- without the screen reader it seems like the selection is still keyboard accessible in Firefox on those platforms. 😕 

Testing NVDA on Chrome and VoiceOver on Chrome and Safari the controls work as expected no problem. 🤦‍♂ We might be encountering a browser bug/quirk here.

### Operating system
Windows 10 and macOS latest

### Browser
Firefox latest

### Assistive technology used to verify
NVDA and VoiceOver

## Steps to reproduce the issue

1. go to http://react.carbondesignsystem.com/iframe.html?id=multiselect--default
2. using NVDA, VoiceOver, or possibly JAWS navigate to the multiselect and open the options menu
3. attempt to focus the menu/any menu item

**Comment by dakahn** [comment]
closed by https://github.com/carbon-design-system/carbon/pull/5373
